src/components/__tests__/data-table.spec.jsx

```diff
@@ -31,7 +31,7 @@ describe('DataTable', () => {
     },
   ];
   const data = fillArray(10, (element, index) => ({
-    [idKey]: `id-${index}`,
+    [idKey]: `id${index}`,
```

Comment:
Was there any specific reasons to remove the `-` from `id-${index}`?

Reply:
Yes, because of the "selected" one above.